src/ServiceControl.UnitTests/ApprovalFiles/APIApprovals.PlatformSampleSettings.approved.txt

```diff
@@ -17,25 +17,25 @@
   "DisableRavenDBPerformanceCounters": true,
   "SkipQueueCreation": false,
   "RunCleanupBundle": false,
-  "RootUrl": "http://localhost:8888/",
+  "RootUrl": "http://localhost:33333/",
```

Comment:
Why his this whole file be changed? I'm pretty sure originally we set explicit values for the test. The only thing that should be changed in this file is the AuditRentionPeriod right?

Reply:
There is something weird going on here. I reverted https://github.com/Particular/ServiceControl/pull/1691/commits/76f4de67b659c692e17e521093e0f31085085d40 locally and ran into this approval test failing on the RootUrl. I looked and the appSettings that in the unit test folder are nothing like the ones in the app.config of the unit tests (and I'm not entirely sure how they ever were). We also have some tests that explicitly write to the ServiceControl.exe.config file and then make assertions about their state, so there is a timing issue there.

Reply:
Ah. Because we don't use the ServiceControl.exe.config ones. We use the ServiceControl.UnitTests.dll.config.
We still destroy that file in a later test and it's not recopied (because the one in the bin folder is newer) on a build. Anyway, that last commit is wrong. What's the best way forward? Force-push after a revert?